compute_tools/src/http/api.rs

```diff
@@ -62,6 +63,18 @@ async fn routes(req: Request<Body>, compute: &Arc<ComputeNode>) -> Response<Body
             Response::new(Body::from(serde_json::to_string(&metrics).unwrap()))
         }
 
+        // Prometheus metrics
+        (&Method::GET, "/metrics") => {
+            info!("serving /metrics GET request");
```

Comment:
Could we use `debug!` here? There would be a lot of not-so-useful logs produced in practice, as metrics are collected rather frequently.

Reply:
fixed